src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -111,6 +117,17 @@ private static String getLocalAppDataLocation() {
     return possiblePaths;
   }
 
+  @VisibleForTesting
+  static String unquote(String path) {
+    if (path.startsWith("\"")) {
```

Comment:
Absent normative documentation, I don't know that it is. The " is always illegal in paths on Windows, even if there's only one.